viewing_party/party.py

```diff
@@ -0,0 +1,156 @@
+
+#Test Wave 01 functions
+
+#Function returns a dictionary containing movie details
+def create_movie(movie_title, genre, rating):
+    if movie_title and genre and rating:
+        movie={
+            "title": movie_title,
+            "genre": genre,
+            "rating" : rating
+        }
+        return movie
+    else:
+        return None
```

Comment:
Consider first testing for the edge case, and then moving the main functionality of your function outside the `if/else`. This programming pattern is called a guard clause and can among other things enhance the readability of your code.
 
```suggestion
    if not movie_title and not genre and not rating:
        return None
        
    movie={
            "title": movie_title,
            "genre": genre,
            "rating" : rating
        }
        return movie
```